xs/src/libslic3r/Config.cpp

```diff
@@ -91,28 +85,28 @@ void
 Config::set(const t_config_option_key& opt_key, const std::string& value)
 {
     try {
-        const auto& def {print_config_def.options.at(opt_key)};
+        const auto& def = print_config_def.options.at(opt_key);
```

Comment:
`auto foo = { xx }` does not work on gcc 4.9 because the type of `foo` is not the right type but always `std::initializer_list<T>`, so the first Travis build fails. I think this is because gcc 4.9 supports C++14 but not C++17 which made this smarter.
This PR removes the `#ifndef SLIC3RXS` guards, so some C++17 code is now exposed to gcc 4.9 on Travis. I felt more comfortable with changing many `auto foo = { xx }` lines than changing the Travis build chain which has cached dependencies and other joys.